app/Notifications/UsernameReminder.php

```diff
@@ -13,21 +13,39 @@
 
 namespace App\Notifications;
 
+use Illuminate\Translation\Translator;
+use Illuminate\Contracts\Config\Repository;
+use Illuminate\Routing\UrlGenerator;
 use Illuminate\Bus\Queueable;
 use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Notifications\Messages\MailMessage;
 use Illuminate\Notifications\Notification;
 
-class UsernameReminder extends Notification implements ShouldQueue
+final class UsernameReminder extends Notification implements ShouldQueue
 {
     use Queueable;
+    /**
+     * @var \Illuminate\Translation\Translator
+     */
+    private $translator;
+    /**
+     * @var \Illuminate\Contracts\Config\Repository
+     */
+    private $configRepository;
+    /**
+     * @var \Illuminate\Routing\UrlGenerator
+     */
+    private $urlGenerator;
 
     /**
      * Create a new notification instance.
      *
      * UsernameReminderEmail constructor.
      */
-    public function __construct()
+    public function __construct(Translator $translator, Repository $configRepository, UrlGenerator $urlGenerator
```

Comment:
Missing closing bracket and opening brace.

Reply:
Fixed. Thank you.